combat-trainer.lic

```diff
@@ -2645,6 +2658,7 @@ class AttackProcess
   end
 
   def attack_thrown(game_state)
+    return if DRRoom.dead_npcs.size > 0
```

Comment:
❓ Won't this abort throwing if there is a dead NPC and a living NPC? Why was this change made?

Reply:
Yes, helps to prevent empaths from attacking a live npc if their current target dies before the loot check.

Reply:
maybe make the new return ```return if DRRoom.dead_npcs.size > 0 && DRStats.empath?``` to limit any potential breakage to empaths.

Reply:
Might be better to call the check face instead if dead Npc and empath